src/tests/ov_helpers/ov_snippets_models/src/subgraph_mlp.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "subgraph_mlp.hpp"
+#include "openvino/opsets/opset15.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+std::shared_ptr<ov::Model> MLPSeqFunction::initOriginal() const {
```

Comment:
Several suggestions regarding builder:
1. `B` and `add` inputs must be constant (tokenization pass has such requirements)
2. Please cover the case with FakeQuantize node on MatMul A input and low precision B input (maybe it is better to do it in a separate builder class)
3. Can we add a parameter which will determine how many [FQ->Matmul->Eltwise] sequences should be created? Only 1 matmul is not enough to fully cover the tokenization pass. Moreover, only one matmul is not tokenized (as I can see from the pass logic)
4. Please add Softmax after all Matmul sequences